lisp/ui/widgets/dbmeter.py

```diff
@@ -143,7 +183,26 @@ def paintEvent(self, e):
             )
             painter.drawPixmap(decayRect, self._pixmap, decayRect)
 
+            # Draw markings
+            x_start = meterRect.x() + meterRect.width() / 2
+            x_end = meterRect.x() + meterRect.width()
+            for mark in self._markings:
+                painter.drawLine(QPointF(x_start, mark[0]), QPointF(x_end, mark[0]))
+
             # Move to the next meter
             meterRect.translate(meterWidth, 0)
 
+        # Write the scale marking text
+        text_height = QFontMetrics(self.font()).height()
+        text_offset = text_height / 2
+        painter.setPen(self.palette().windowText().color())
+        painter.drawText(0, 0, width, text_height, Qt.AlignRight, str(self.dBMax))
```

Comment:
We can account for the empty space on top of the character, using (as before) `QFontMetrics.descent()`
```suggestion
        painter.drawText(0, -QFontMetrics(self.font()).descent(), width, text_height, Qt.AlignRight, str(self.dBMax))
```

Reply:
This exact change would have the sole result of drawing the top half of the value of `self.dBMax` off the top of the widget, clipping it.

Reply:
:thinking: I get a different result, for some reason I've a larger gap at the top, probably depends a lot from the used font, better not to use it